npm/README.md

```diff
@@ -0,0 +1,114 @@
+# electron-prebuilt
+
+[![Travis build status](http://img.shields.io/travis/electron-userland/electron-prebuilt.svg?style=flat)](http://travis-ci.org/electron-userland/electron-prebuilt)
```

Comment:
Maybe we can just take this README out completely, and use the one from root? That way we can keep the docs on the npm package page consistent with what we have up on the github repo.